src/Kunstmaan/FormBundle/Helper/FormMailer.php

```diff
@@ -16,18 +16,18 @@ class FormMailer implements FormMailerInterface
     /** @var \Swift_Mailer */
     private $mailer;
 
-    /** @var \Symfony\Bundle\TwigBundle\TwigEngine */
+    /** @var DelegatingEngine */
     private $templating;
 
     /** @var \Symfony\Component\DependencyInjection\ContainerInterface */
     private $container;
 
     /**
      * @param Swift_Mailer       $mailer     The mailer service
-     * @param TwigEngine         $templating The templating service
+     * @param DelegatingEngine   $templating The templating service
      * @param ContainerInterface $container  The container
      */
-    public function __construct(Swift_Mailer $mailer, TwigEngine $templating, ContainerInterface $container)
+    public function __construct(Swift_Mailer $mailer, DelegatingEngine $templating, ContainerInterface $container)
```

Comment:
I think it's best to typehint against `Symfony\Component\Templating\EngineInterface` so we don't lock to that deligatingEnigine class